dietpi/dietpi-software

```diff
@@ -5293,6 +5304,34 @@ Package: wireguard wireguard-dkms wireguard-tools\nPin: release n=bullseye\nPin-
 
 		fi
 
+		software_id=177 # Firefox Sync Server
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			# Dependencies
+			DEPS_LIST='python-virtualenv libffi-dev libssl-dev'
+			# - Pre-compiling required on ARM
+			(( $G_HW_ARCH < 10 )) && DEPS_LIST+=' libffi-dev libssl-dev'
+
+			INSTALL_URL_ADDRESS='https://github.com/mozilla-services/syncserver'
+
+			# If there is an existing installation update it as per Mozilla's instructions, else fresh install
+			if [[ -d /etc/firefox-sync ]]; then
```

Comment:
Generally /etc is not intended for programs, but for config files only. What do you think about using `/opt/firefox-sync`, or if it contains variable config files that cannot be placed into a different dir, `$G_FP_DIETPI_USERDATA/firefox-sync`?

Generally best DietPi practice would be to have the program dir, that can be fully purged on reinstall in /opt and all config files in $G_FP_DIETPI_USERDATA, but not possible in all cases.